cnd/src/main.rs

```diff
@@ -4,18 +4,18 @@ use btsieve::{bitcoin::BitcoindConnector, ethereum::Web3Connector};
 use cnd::{
     comit_i_routes,
     config::{self, Settings},
-    http_api::{self, route_factory},
+    connector::{Connector, Dependencies},
+    http_api::route_factory,
     network::{self, Network},
     seed::Seed,
     swap_protocols::{
-        self,
         metadata_store::{InMemoryMetadataStore, MetadataStore},
         rfc003::{
-            alice::AliceSpawner,
-            bob::BobSpawner,
+            alice::{InitiateRequest, SendRequest},
+            bob::SpawnBob,
```

Comment:
:thinking: Alice has nice trait names that have meaning within the protocol, but Bob doesn't. Would it be better to rename `SpawnBob` to `AcceptRequest`, for example?

Reply:
Good point than 'spawn' does not match the other names.  I'm leaving this for further consideration on the 'remove code duplication in spawners ticket'.  Will add a comment there about this.  https://github.com/comit-network/comit-rs/issues/1610